addons/mail/models/ir_http.py

```diff
@@ -13,7 +13,10 @@ def session_info(self):
         """Override to add the current user data (partner or guest) if applicable."""
         result = super().session_info()
         store = StoreData()
-        self.env["res.users"]._init_store_data(store)
+        default_user = self.env["res.users"]
```

Comment:
I'm not sure why `default_` in the name?
Some ideas: `user_with_context`, `ResUsers`, `User`

Reply:
because i did not have any other idea :smile: . I'll go for ResUsers if you like it better.